tools/kdump_configure.sh

```diff
@@ -0,0 +1,351 @@
+    check_exit_status "apt_get_install $package_name"
+}
+#######################################################################
+#
+# yum_install()
+#
+#######################################################################
+function yum_install ()
+{
+    package_name=$1
+    yum install -y $package_name
+    check_exit_status "yum_install $package_name"
+}
+#######################################################################
+#
+# zypper_install()
+#
+#######################################################################
+function zypper_install ()
+{
+    package_name=$1
+    zypper --non-interactive in $package_name
+    check_exit_status "zypper_install $package_name"
+}
+#######################################################################
+#
+# install_package()
+#
+#######################################################################
+function install_package ()
+{
+    local package_name=$@
+
+    if [ `which yum` ]; then
+    	yum_install "$package_name"
+    elif [ `which apt-get` ]; then
+    	apt_get_install "$package_name"
+	elif [ `which zypper` ]; then
+		zypper_install "$package_name"
+    fi
+}
+#######################################################################
+#
+# config_kdump_RHEL()
+#
+#######################################################################
+function config_kdump_RHEL
+{
+    LogMsg "Configuring Kdump on RHEL60"
+    if [ x$host_version == "x" ]; then
+		$host_version=`dmesg | grep "Host Build" | sed "s/.*Host Build://"|sed "s/^.*-\([0-9]*\.[0-9]*\)-.*/\1 /"`
+		if [ x$host_version == "x" ]; then
+			LogMsg "Unable to find Host version"
+			UpdateSummary "ABORTED"
+		fi
+	fi
+    LogMsg "Installing required packages"
+    install_package "kexec-tool crash"
+    local distro_version=`detect_linux_ditribution_version`
+    if [ $distro_version == "6.0" ]; then
+        if [ $host_version == "6.2" ]; then
+
+            LogMsg "Configuring Kdump for `detect_distribution`-$distro_version VM running on `get_host_version $host_version`"
+            sed -i "s/\(^default.*\)/#\1/" /etc/kdump.conf
+            LogMsg "/etc/kdump.conf 'default' action is commented to boot from 'initrd-...kdump.img'."
+
+            if [ `cat /boot/grub/grub.conf | grep -v "#"| grep kernel.*root=| grep "crashkernel=auto" | wc -l` == 0 ]; then
+                sed -i "s/\(kernel.*root.*\)/\1 crashkernel=auto/" /boot/grub/grub.conf
+                LogMsg "Configured '/boot/grub/grub.conf' with 'crashkernel=auto'"
+            else
+                LogMsg "'/boot/grub/grub.conf' already configured with 'crashkernel=auto' skipping grub configuration..."
+            fi
+        else
+			LogMsg "Unsupported Host version $host_version"
+			UpdateSummary "ABORTED"
+		fi
+		
+    else
+        LogMsg "Unsupported OS version $distro_version"
+        UpdateSummary "ABORTED"
+    fi
+    LogMsg "Kdump configuration is completed. Please reboot VM to apply the changes."

```

Comment:
crashkernel should be 384M minimum , auto does not work. also i do not see where are you modifying config files to blacklist hyperv modules as mentioned in kb.